publication-data.rkt

```diff
@@ -0,0 +1,8 @@
+#lang racket
+
+;; Defines common publication data used in multiple files
+
+(provide (struct-out publication))
+
+;; string string string number (maybe string) -> elem?
+(define-struct publication (title authors venue year link) #:prefab)
```

Comment:
An idea ... if this file provided a constructor just like Scribble's [`make-bib`](http://docs.racket-lang.org/scriblib/autobib.html#%28def._%28%28lib._scriblib%2Fautobib..rkt%29._make-bib%29%29) instead of `(publication ....)`, people using Scribble could copy/paste data from `publications.rkt`.

Like Sam's [`gradual-typing-bib`](https://github.com/samth/gradual-typing-bib/blob/master/main.rkt)

(We could also add "export" links to the website, but that should be a separate PR.)